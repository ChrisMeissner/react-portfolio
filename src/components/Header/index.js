import React from 'react';

const headerStyle = {
  display: 'flex',
  justifyContent: 'center',
  width: '40%',
  height: '100%',
  fontSize: '60px',
  color: 'green'
};

function Header() {
  return (
    <section style={headerStyle}>
      Chris Meissner
    </section>
  )
};

export default Header;