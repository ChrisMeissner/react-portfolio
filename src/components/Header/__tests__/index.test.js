import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from '..';

afterEach(cleanup);

describe('Header Component', () => {
  // renders Header test
  // first test
  it('renders', () => {
    render(<Header/>);
  });

  // second test
  it('matches snapshot DOM node structure', () => {
    // render Header
    const { asFragment } = render(<Header />);
    expect(asFragment()).toMatchSnapshot();
  })

});