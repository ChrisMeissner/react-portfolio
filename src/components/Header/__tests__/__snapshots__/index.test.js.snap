// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div
    style="display: flex; justify-content: center; align-items: center; width: 40%; height: inherit; font-size: 60px; color: rgb(255, 255, 255);"
  >
    Chris Meissner
  </div>
</DocumentFragment>
`;
